samples/demo/src/main/res/layout/activity_main.xml

```diff
@@ -110,8 +110,6 @@
       <com.facebook.drawee.view.SimpleDraweeView
         android:id="@+id/alpha_webp"
         style="@style/drawee"
-        fresco:backgroundImage="@color/transparent"
```

Comment:
Because we no longer need to preallocate space for background/overlay in order to change it dynamically later on, and more importanitly we no longer need to allocate this useless transparent drawable.
Pure win :)